crates/astria-sequencer/src/app.rs

```diff
@@ -74,13 +60,6 @@ impl App {
 
     #[instrument(name = "App:init_chain", skip(self))]
     pub async fn init_chain(&mut self, genesis_state: GenesisState) -> Result<()> {
-        // allocate to hard-coded accounts for testing
-        let mut accounts = genesis_state.accounts;
-        accounts.append(&mut default_genesis_accounts());
-        let genesis_state = GenesisState {
-            accounts,
-        };
-
```

Comment:
this *probably* shouldn't be here, except now the manual test instructions in the PR description/crate README will fail unless this is re-added since genesis parsing isn't implemented yet (#67) thus there are no funded accounts at genesis anymore.

Reply:
not sure what the exact right solution is here, maybe adding a feature flag for a test-genesis-state or something like that which adds this, until #67 is done?

Reply:
Fair point. This is IMO core functionality. I will add it in.

Reply:
Added through a simple clap config, see `cargo run -- --help`